.github/workflows/tld-update.yml

```diff
@@ -0,0 +1,45 @@
+name: tld-update
+on:
+  schedule:
+    # Run every hour, at the 15 minute mark. E.g.
+    # 2020-11-29 00:15:00 UTC, 2020-11-29 01:15:00 UTC, 2020-11-29 02:15:00 UTC
+    - cron:  '15 * * * *'
+jobs:
+  psl-gtld-update:
+    name: Check for TLD data updates
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ^1.15
+
+      - name: Set current date as env variable
+        run: echo "NOW=$(date +'%Y-%m-%dT%H:%M:%S %Z')" >> $GITHUB_ENV
```

Comment:
This is mostly a nit, but I'm curious the choice for $GITHUB_ENV versus using a step output. The main argument against I can think of is that it requires an explicit ID for the step, but were there other considerations?

That is
```
- name: Set the current date as env variable
  id: get-date
  run: echo "::set-output name=now::$(date +'%Y-%m-%dT%H:%M:%S %Z')"
```

and then later using `${{ steps.get-date.now }}`

Not a blocker by any means, and I'm not sure if either approach handles newline injection that well, but that's what made me think of it.

Reply:
> This is mostly a nit, but I'm curious the choice for $GITHUB_ENV versus using a step output. The main argument against I can think of is that it requires an explicit ID for the step, but were there other considerations?

I fiddled with a few options for this when I first implemented it in ZLint and I remember having trouble getting an alternative approach to work but I can't remember the details. Possible it was operator error :-)

I'll try testing a step output based approach to see if I can get it working. It does seem cleaner 👍

Reply:
@sleevi Fixed up in ff6c3ba, thanks for the suggestion :+1: Tested successfully with [another mock PR](https://github.com/cpu/list/pull/3).

> and then later using ${{ steps.get-date.now }}

One small note: using this syntax literally resulted in an empty string (eg [this PR](https://github.com/cpu/list/pull/2)). I had to reference the `now` output ID within a more specific keyspace: `${{ steps.get-date.outputs.now }}`.